ccdaservice/utils/demographics/populate-demographics.mock.js

```diff
@@ -0,0 +1,541 @@
+const MOCK_ADDRESS = require('./previous-addresses.mock').TEST_CASES[0][0];
+const { fDate } = require('../date/date');
+const {
+    NULL_FLAVOR,
+    DECLINED_TO_SPECIFY,
+    NOT_INFORMED,
+} = require('../constants');
+const TODAY = '2021-04-06T18:55:55.301Z';
+
+const mockPatient = {
+    race: 'mock_race',
+    race_group: 'mock_race_group',
+    ethnicity: 'mock_ethnicity',
+    prefix: 'Ms',
+    fname: 'Catherine',
+    mname: 'Zeta',
+    lname: 'Jones',
+    suffix: 'The daughter of dragons',
+    birth_mname: 'Not at all',
+    birth_lname: 'Famous',
+    birth_fname: 'Lilly',
+    dob: '1969-09-25',
+    gender: 'female',
+    uuid: '052698',
+    status: 'married',
+    phone_home: '111-1111',
+    phone_mobile: '222-2222',
+    phone_work: '333-3333',
+    phone_emergency: '444-4444',
+    email: 'cathy@zeta.jones',
```

Comment:
I recommend either using ```@example.com``` or one of the possible sample emails that we know won't resolve instead of us accidentally sending email to someone in the future.

Reply:
```suggestion
    email: 'cathy@example-openemr.com',
```

Got it, thanks for the suggestion!